.gitlab-ci.yml

```diff
@@ -0,0 +1,35 @@
+image: docker:stable
+
+variables:
+  DOCKER_HOST: tcp://docker:2375
+  DOCKER_DRIVER: overlay2
+  BUILDER_TAG: index.docker.io/algorithmiahq/dev-center:latest
+  IMAGE_TAG: index.docker.io/algorithmiahq/dev-center:$CI_COMMIT_SHA
+
+services:
+  - docker:18.09.7-dind
+
+stages:
+  - build
+
+#
+# Build
+#
+
+quick-build:
```

Comment:
:bulb: Any reason we couldn't simply call this "build", since it's the only possible build step?

Reply:
Updated!